### ISSUE TITLE
Not having same reset tools as BTCPayserver

### ISSUE THREAD

**Issue by 2pac1** [btcpayserver/btcTransmuter#55]
Hi,  

 Once you get locked out with the first admin user, there should be a way for one to manage to get access if he/she has access to the server.

Tnx guys!

**Comment by Eskyee** [comment]
<img width="1153" alt="Screenshot 2020-05-07 at 01 10 02" src="https://user-images.githubusercontent.com/30862925/81240575-a40d7800-8fff-11ea-87fc-e7397084b2b1.png">

I have also noticed no lost password for the end user ??? forgot your password , can we have this added ?? @Kukks 

Also once you get locked out with the first admin user, there should be a way for one to manage to get access if he/she has access to the server.

I also have noticed this, ?? how does one reset the admin password, if forgotten. cheers

**Comment by bigsk1** [comment]
Agreed, had same issue. I installed transmuter like a year ago, some update must have removed it, a year later just revisiting it, had to install again and was no register option. Had to dig deep but found password.
